<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AAVS1</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1061</width>
     <height>651</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_map">
    <attribute name="title">
     <string>AAVS1 Map</string>
    </attribute>
    <widget class="QWidget" name="plotWidgetMap" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>690</width>
       <height>580</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_polx">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pol X</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_poly">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pol Y</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_base">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>140</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Antenna Base #</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_tpm">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>170</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TPM #</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_not">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>260</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Cleaned</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_single">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>290</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Single Cleaned</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_double">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>320</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Double Cleaned</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_ferrite">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>350</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ferrite</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_gnd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>380</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>GND Jacket</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_condom">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>410</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Condom</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_capacitor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>440</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacitor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_feed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>470</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>No Feedthrough</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_rms">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RMS</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_dBm">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>dBm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_runMap">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>520</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_saveMap">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>520</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_group">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>230</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Group by:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_conf">
    <attribute name="title">
     <string>Configuration</string>
    </attribute>
    <widget class="QGroupBox" name="groupAntennaList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>1021</width>
       <height>341</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>AAVS1 Antenna List</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QScrollArea" name="qscroll_aavs">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>1001</width>
        <height>281</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>999</width>
         <height>279</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_N">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>#</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Base</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>HC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rox</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TPM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_6">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>RX</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fibre</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Colour</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_9">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rib</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_10">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>241</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FLAGS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupAntennaList_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1021</width>
       <height>191</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Plot List</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QScrollArea" name="qscroll_plot">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>1001</width>
        <height>151</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>999</width>
         <height>149</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="qbutton_UpdatePlot">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>210</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Plot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_DeleteList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_LoadList">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Load List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_SaveList">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save List</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_plot">
    <attribute name="title">
     <string>Plot Spectra</string>
    </attribute>
    <widget class="QWidget" name="plotWidgetMap_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>781</width>
       <height>491</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_ymin">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>570</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y MIN:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_ymax">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>570</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y MAX:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="qtext_xmin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>570</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="qtext_ymax">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>570</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="qtext_ymin">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>570</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>-100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_xmax">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>570</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X MAX:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="qtext_xmax">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>570</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>400</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_xmin">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>570</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X MIN:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_plotSave">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>520</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qbutton_plotStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
